LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013, Alexander Miertsch contact@prooph.de
+Copyright (c) 2017, Alexander Miertsch contact@prooph.de
```

Comment:
should be:

Copyright (c) 2013-2017, prooph software GmbH
Copyright (c) 2015-2017, Sascha-Oliver Prolic